contracts/upgradeable_contracts/erc20_to_erc20/ForeignBridgeExtendedErcToErc.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity 0.4.24;
+
+
+import "./ForeignBridgeErcToErc.sol";
+import "../ERC20ExtendedBridge.sol";
+
+
+contract ForeignBridgeExtendedErcToErc is ERC20ExtendedBridge, ForeignBridgeErcToErc {
+
+    event UserRequestForAffirmation(address recipient, uint256 value);
+
+    function extendedInitialize(
+        address _validatorContract,
+        address _erc20token,
+        uint256 _requiredBlockConfirmations,
+        uint256 _gasPrice,
+        uint256 _dailyLimit,
+        uint256 _maxPerTx,
+        uint256 _minPerTx,
+        uint256 _homeDailyLimit,
+        uint256 _homeMaxPerTx,
+        address _owner
+    ) public returns(bool) {
+        require(_minPerTx > 0 && _maxPerTx > _minPerTx && _dailyLimit > _maxPerTx);
+        uintStorage[keccak256(abi.encodePacked("dailyLimit"))] = _dailyLimit;
+        uintStorage[keccak256(abi.encodePacked("minPerTx"))] = _minPerTx;
+
+        _initialize(
+            _validatorContract,
+            _erc20token,
+            _requiredBlockConfirmations,
+            _gasPrice,
+            _maxPerTx,
+            _homeDailyLimit,
+            _homeMaxPerTx,
+            _owner
+        );
+        return isInitialized();
+    }
+
+    function initialize(
+        address /* _validatorContract */,
+        address /* _erc20token */,
+        uint256 /* _requiredBlockConfirmations */,
+        uint256 /* _gasPrice */,
+        uint256 /* _maxPerTx */,
+        uint256 /* _homeDailyLimit */,
+        uint256 /* _homeMaxPerTx */,
+        address /* _owner */
+    ) public returns(bool) {
+        revert();
```

Comment:
there is no need to make this if the method `extendedInitialize` is called `initialize` instead. What do you think?